usage: train.py [-h] [--local_rank LOCAL_RANK] [--run_num RUN_NUM]
                [--yaml_config YAML_CONFIG] [--config CONFIG]
usage: train.py [-h] [--local_rank LOCAL_RANK] [--run_num RUN_NUM]
                [--yaml_config YAML_CONFIG] [--config CONFIG]
usage: train.py [-h] [--local_rank LOCAL_RANK] [--run_num RUN_NUM]
                [--yaml_config YAML_CONFIG] [--config CONFIG]
train.py: error: unrecognized arguments: ./config/UNet_transpose.yaml
train.py: error: unrecognized arguments: ./config/UNet_transpose.yaml
train.py: error: unrecognized arguments: ./config/UNet_transpose.yaml
usage: train.py [-h] [--local_rank LOCAL_RANK] [--run_num RUN_NUM]
                [--yaml_config YAML_CONFIG] [--config CONFIG]
usage: train.py [-h] [--local_rank LOCAL_RANK] [--run_num RUN_NUM]
                [--yaml_config YAML_CONFIG] [--config CONFIG]
usage: train.py [-h] [--local_rank LOCAL_RANK] [--run_num RUN_NUM]
                [--yaml_config YAML_CONFIG] [--config CONFIG]
train.py: error: unrecognized arguments: ./config/UNet_transpose.yaml
usage: train.py [-h] [--local_rank LOCAL_RANK] [--run_num RUN_NUM]
                [--yaml_config YAML_CONFIG] [--config CONFIG]
train.py: error: unrecognized arguments: ./config/UNet_transpose.yaml
train.py: error: unrecognized arguments: ./config/UNet_transpose.yaml
train.py: error: unrecognized arguments: ./config/UNet_transpose.yaml
usage: train.py [-h] [--local_rank LOCAL_RANK] [--run_num RUN_NUM]
                [--yaml_config YAML_CONFIG] [--config CONFIG]
train.py: error: unrecognized arguments: ./config/UNet_transpose.yaml
*****************************************
Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
*****************************************
Traceback (most recent call last):
  File "/opt/python/cp37-cp37m/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/opt/python/cp37-cp37m/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/opt/python/cp37-cp37m/lib/python3.7/site-packages/torch/distributed/launch.py", line 263, in <module>
    main()
  File "/opt/python/cp37-cp37m/lib/python3.7/site-packages/torch/distributed/launch.py", line 259, in main
    cmd=cmd)
subprocess.CalledProcessError: Command '['/opt/python/cp37-cp37m/bin/python', '-u', 'train.py', '--local_rank=7', '--run_num=14', '--yaml_config=', './config/UNet_transpose.yaml', '--config=multi8']' returned non-zero exit status 2.
srun: error: cgpu07: task 0: Exited with exit code 1
srun: Terminating job step 838853.0
Wed Jul 22 12:48:09 PDT 2020
